### ISSUE TITLE
Share dialog: no confirmation

### ISSUE THREAD

**Issue by tobiasKaminsky** [owncloud/android#1555]
When sharing a file/folder with add an user or create a link.
But then...? You have to click back to close this dialog.
"Normally" back is something to undo/revert/abort.
This is somehow confusing, at least for me ;)

@jancborchardt @AndyScherzinger

**Comment by tobiasKaminsky** [comment]
What about two buttons on the bottom:
- unshare: is shown if link / user is set and unshares completly
- done: just closes the dialog

**Comment by AndyScherzinger** [comment]
Yeah, this has also been confusing to me, so unshare and done or OK are fine I think. Especially the unshare makes sense because right now unshare means one has to remove all users and use the switch to deativate the link...

**Comment by tobiasKaminsky** [comment]
ping @jancborchardt

**Comment by DeepDiver1975** [comment]
🤖 Automated issue triage by Claude Code review agent.

This issue was opened before 2020 and has had no activity since then, so we are
closing it as outdated. If this is still relevant on a current ownCloud release,
please **reopen** (or open a fresh issue) with up-to-date details — thank you\!
